src/app/main.cpp

```diff
@@ -113,6 +113,24 @@ typedef SInt32 SRefCon;
 #include "qgsopenclutils.h"
 #endif
 
+/**
+ * Print QGIS version
+ */
+void version( )
+{
+  const QString msg = QStringLiteral( "QGIS %1 '%2' (%3)\n" ).arg( VERSION ).arg( RELEASE_NAME ).arg( QGSVERSION );
+
+#ifdef Q_OS_WIN
+  MessageBox( nullptr,
```

Comment:
Is a message box on windows expected behavior?
Doesn't this mean it cannot be used by scripts?

Reply:
@m-kuhn I've kept this behaviour. It was introduced by @jef-n https://github.com/qgis/QGIS/commit/f61c993e1355cd8ded491e48bdeb3b0aea939c34 3 years ago.

I would prefer to have the output written to `stdout`. Let's see if @jef-n has something to add about this behaviour.

This is the current behaviour in Windows is shown below:

![2019-06-14](https://user-images.githubusercontent.com/163681/59511219-a29a0500-8ead-11e9-9335-a337fc873ae1.png)

Reply:
@m-kuhn Considering again this behaviour, it is easier to argue that the `--help` can be displayed on a proper window, since this `--help` output is to be read by humans.

The output of `--version` is more useful for scriptig. I'll change the code.

Thank you for your comment.

Reply:
Done!